### DISCUSSION TITLE
Resume building query

### DISCUSSION THREAD

**Discussion by LordAdhaar** [Cyfrin/foundry-full-course-cu#3040] [General]
Recently finished Defi Protocol and DAO, project.
I like these projects and they align with the reasons I wanted to learn web3. 
I want to add them to resume so that I can apply for jobs.
How do you add a smart contract project to your resume?
Do you just add the link for github repo along with the address of the deployed smart contract?

**Comment by EngrPips** [comment]
Hey @LordAdhaar, I believe doing that is sufficient, but I will advise you to build something entirely different yourself, and it might be based on the idea of the Defi Protocol and the Dao project. Show the recruiter that you can do stuff yourself without following a tutorial.

**Reply by heheboii11** [reply]
Yes there are many dev who are having the same projects into their portfolio, you need to show something new and innovative. You have learnt so much by completing this course but you need to implement them and build your own projects so that you can show your skills to them
